ui/plugins/pinboard.js

```diff
@@ -245,5 +249,36 @@ treeherder.controller('PinboardCtrl', [
         $scope.pinnedJobs = thPinboard.pinnedJobs;
         $scope.relatedBugs = thPinboard.relatedBugs;
 
+        var getRevisionTips = function(projectName, list) {
```

Comment:
When I am trying to call this same function, data is not loaded properly. After this function is executed, then only data is available in the list, i.e., javascript is calling the function asynchronously. How can I solve this issue?